src/icons/microsoft-office/IconOffice.tsx

```diff
@@ -0,0 +1,104 @@
+import * as React from 'react';
+import uniqueId from 'lodash/uniqueId';
+
+import AccessibleSVG from '../accessible-svg';
+
+import { TwoTonedIcon } from '../iconTypes';
+
+const ICON_CLASS = 'icon-office';
+
+class IconOffice extends React.Component<TwoTonedIcon> {
```

Comment:
Is this the current Office logo and approved by design? The other Office SVGs use the flat design style: https://opensource.box.com/box-ui-elements/#/Icons?id=iconofficewordmark

Reply:
Thank you for pointing this out! I've replaced Office icon with flat one to mach current wordmark icon